src/mame/technics/kn5000.cpp

```diff
@@ -0,0 +1,336 @@
+// license:GPL2+
+// copyright-holders:Felipe Sanches
+/******************************************************************************
+
+	Technics SX-KN5000 music keyboard driver
+
+******************************************************************************/
+
+#include "emu.h"
+#include "bus/technics/hdae5000.h"
+#include "cpu/tlcs900/tmp94c241.h"
+#include "imagedev/floppy.h"
+#include "machine/gen_latch.h"
+#include "machine/upd765.h"
+#include "screen.h"
+#include "sound/beep.h"
+#include "speaker.h"
+#include "video/pc_vga.h"
+
+
+namespace {
+
+class kn5000_state : public driver_device
+{
+public:
+	kn5000_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_subcpu(*this, "subcpu")
+		, m_fdc(*this, "fdc")
+		, m_CHECKING_DEVICE_LED_CN11(*this, "checking_device_led_cn11")
```

Comment:
Is it absolutely necessary to have UPPERCASE for this specific device? What's the rationale here?

Reply:
No, it's not necessary. I'll change it.